src/main/java/mate/project/dto/CreateBookRequestDto.java

```diff
@@ -1,11 +1,18 @@
 package mate.project.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotNull
         String title,
+        @NotNull
         String author,
+        @NotNull
         String isbn,
+        @NotNull
+        @Min(0)
         BigDecimal price,
         String description,
```

Comment:
The 'description' field of the CreateBookRequestDto class is not validated. Consider adding appropriate validation constraints to this field.